From d2afd20e4257d36e14ef6e9bc773011cda57f82c Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Sun, 27 Dec 2015 16:22:17 +0200
Subject: [PATCH 19/30] ARM: i.MX6UL: update cl-som-imx6ul defconfig for
 mcs7830

Enable mcs7830 in defconfig.
This allows using an nfs root in a non-FEC configuration.

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 arch/arm/configs/cl_som_imx6ul_defconfig | 10 +++++++++-
 1 file changed, 9 insertions(+), 1 deletion(-)

diff --git a/arch/arm/configs/cl_som_imx6ul_defconfig b/arch/arm/configs/cl_som_imx6ul_defconfig
index c9bb3f1..d8b63bb 100644
--- a/arch/arm/configs/cl_som_imx6ul_defconfig
+++ b/arch/arm/configs/cl_som_imx6ul_defconfig
@@ -162,8 +162,16 @@ CONFIG_MICREL_PHY=y
 CONFIG_USB_PEGASUS=m
 CONFIG_USB_RTL8150=m
 CONFIG_USB_RTL8152=m
-CONFIG_USB_USBNET=m
+CONFIG_USB_USBNET=y
+CONFIG_USB_NET_AX8817X=m
+CONFIG_USB_NET_AX88179_178A=m
+CONFIG_USB_NET_CDCETHER=m
 CONFIG_USB_NET_CDC_EEM=m
+CONFIG_USB_NET_CDC_NCM=m
+CONFIG_USB_NET_NET1080=m
+CONFIG_USB_NET_MCS7830=y
+CONFIG_USB_NET_CDC_SUBSET=m
+CONFIG_USB_NET_ZAURUS=m
 CONFIG_ATH_CARDS=y
 CONFIG_ATH6KL=m
 CONFIG_ATH6KL_SDIO=m
-- 
1.9.1

